cpp/src/arrow/dataset/file_parquet_encryption_test.cc

```diff
@@ -188,8 +286,13 @@ TEST_F(DatasetEncryptionTest, WriteReadDatasetWithEncryption) {
   AssertTablesEqual(*combined_table, *table_);
 }
 
-// Read a single parquet file with and without decryption properties.
-TEST_F(DatasetEncryptionTest, ReadSingleFile) {
+void read_single_file(
```

Comment:
Please, let's follow the style conventions. Function names should be `CamelCase`.